docs/reference/TiffTags.rst

```diff
@@ -22,7 +22,7 @@ metadata tag numbers, names, and type information.
 
      :param value: Integer Tag Number
      :param name: Tag Name
-     :param type: Integer type from :py:attr:`PIL.TiffTags.TYPES`
+     :param type: Integer type from :py:data:`PIL.TiffTags.TYPES`
```

Comment:
The original `PIL.TiffTags.TYPES` is linked, the new one is not:

* https://pillow--4697.org.readthedocs.build/en/4697/reference/TiffTags.html
* https://pillow.readthedocs.io/en/stable/reference/TiffTags.html

Reply:
This is the same issue as above. While looking into in, I have found the [nitpicky (`-n`)](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-nitpicky) option, which finds additional ~470 broken references. I'll try to take a look at these later if I can; most of them look fairly straightforward to fix.

Reply:
Wow, that's a lot! It might be an idea to do some of those in separate PRs, to make things easier to review/merge. Perhaps by page or by section or something.

Reply:
I've already got it down to 70. There a few issues repeated many times and easily fixed with a few greps or simple changes, the more complicated bits can be done later.